databases/week1/homework1.sql

```diff
@@ -0,0 +1,62 @@
+-- 1. Select the names and phones of all users;
+
+SELECT id, phone FROM user;
+
+-- 2. Select the name of the user with id=10;
+
+SELECT * FROM user WHERE id=10;
```

Comment:
Here as you need to select names, you could just get `name` instead of everything (`*`) 🤓

Reply:
Need to pay more attention :)